pkg/providers/securitygroup/securitygroup.go

```diff
@@ -55,7 +55,11 @@ func (p *Provider) List(ctx context.Context, nodeTemplate *v1alpha1.AWSNodeTempl
 	p.Lock()
 	defer p.Unlock()
 	// Get SecurityGroups
-	securityGroups, err := p.getSecurityGroups(ctx, p.getFilters(nodeTemplate))
+	filters := p.getFilters(nodeTemplate)
```

Comment:
Can you throw a TODO to revisit this for #1327, where we're removing custom LTs and security group will be required.